src/client/datascience/notebook/vscodeNotebookController.ts

```diff
@@ -476,7 +480,7 @@ function getKernelConnectionCategory(kernelConnection: KernelConnectionMetadata)
     switch (kernelConnection.kind) {
         case 'connectToLiveKernel':
             return DataScience.kernelCategoryForJupyterSession();
-        case 'startUsingKernelSpec':
+        case 'startUsingLocalKernelSpec':
```

Comment:
Shouldn't the remote kernelspec be listed here too?

Reply:
Oh, I thought I added it, must have remvoed it, I ermember testing this damn part

Reply:
Fixed